### DISCUSSION TITLE
Multiple takeouts?

### DISCUSSION THREAD

**Discussion by meemkade** [simulot/immich-go#520] [Q&A] [answered]
I exported my google photos, and chose ZIP and the highest file size, that being 50 GB.
The total export was ~114 GB from photos in total. When I used the tool to upload all 3, this is what immich shows:
![image](https://github.com/user-attachments/assets/3d4cd53f-6260-4c18-a32f-f39e5ebbffc5)
Im a bit confused on how im supposed to upload multiple takeouts, do I do it like this, which is what I ended up doing and how im here?:
`immich-go.exe -server http://192.168.10.12:2283 -key redacted upload -google-photos takeout-20241031T062333Z-001.zip takeout-20241031T062333Z-002.zip takeout-20241031T062333Z-003.zip`
Or do I do it this way, one at a time?:
`immich-go.exe -server http://192.168.10.12:2283 -key redacted upload -google-photos takeout-20241031T062333Z-001.zip`
(continued for the next 2 zips)

**Comment by simulot** [comment, marked as answer]
The easiest way to process all zips at the same time is to use this command:

```
immich-go.exe -server http://192.168.10.12:2283 -key redacted upload -google-photos takeout-*.zip
```
It's not a problem to reprocess the file you have already processed.

**Reply by meemkade** [reply]
In the meantime, I already uploaded all files by manually specifying them, should I use your command and wipe what I uploaded or is what I did fine?

**Reply by simulot** [reply]
You're probably pretty good. Therefore, the problem when treating parts one at a time is that some images are left over because relative metadata file may be located in the other part.  The default immich-go behavior is to skip them.

You can re-run the command to treat all parts together. The left over files will be loaded, other simply ignored.
